TwitchChannelPointsMiner/classes/Twitch.py

```diff
@@ -231,7 +246,10 @@ def post_gql_request(self, json_data):
                 headers={
                     "Authorization": f"OAuth {self.twitch_login.get_auth_token()}",
                     "Client-Id": CLIENT_ID,
+                    "Client-Integrity": self.post_integrity(),
+                    "Device-ID": self.device_id,
                     "User-Agent": self.user_agent,
+                    "X-Device-Id": self.device_id,
```

Comment:
Here twitch actually uses
* Authorization
* Client-Integrity: <from /integrity call>
* Client-Id: <same as /integrity call>
* Client-Session-Id: <same as /integrity call>
* Client-Version: <same as /integrity call>
* X-Device-Id: <same as /integrity call>

Reply:
- Add dynamically update `Client-Version` based on @RakSrinaNa suggestion [dae9c70](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/pull/594/commits/dae9c709862ea386e866ea83ef55bc0c70c5999f)
- Add generated `Client-Session-Id` [8594d54](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/pull/594/commits/8594d54f08eec723cc4bc238aa46412f4283df71)

Reply:
@RakSrinaNa Is the `Device-ID` header required?

Reply:
I'd say no as Twitch doesn't uses it.

Reply:
@bossoq I think the last thing outstanding is to remove the `Device-ID` header.

Reply:
Already remove in latest commit.